medusa/providers/torrent/json/torrentleech.py

```diff
@@ -124,6 +124,10 @@ def search(self, search_strings, age=0, ep_obj=None, **kwargs):
                 num_found = json.get('numFound', 0)
                 per_page = json.get('perPage', 35)
 
+                if per_page < 100 and num_found > per_page:
+                    log.info('It is recommended to change "Default Results Per Page" to 100'
+                             ' in your profile options on {name}.', name=self.name)
```

Comment:
Encourages users to set their site profile settings to 100 results per page.
It can't be tweaked in the search url, only on the site.